hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -306,6 +306,13 @@ void OtbnModel::edn_urnd_step(svLogicVecVal *edn_urnd_data /* logic [31:0] */) {
   iss->edn_urnd_step(edn_urnd_data->aval);
 }
 
+void OtbnModel::set_keymgr_value(svLogicVecVal *key0 /* logic [31:0] */, 
+                                 svLogicVecVal *key1, bool valid) {
```

Comment:
I feel like mypy might complain about this part

```suggestion
                                 svLogicVecVal *key1 /* logic [31:0] */,
                                 bool valid) {
```